cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/BaseController.java

```diff
@@ -0,0 +1,26 @@
+package cloud.foundry.cli.services;
+
+import picocli.CommandLine;
+
+/**
+ * This class works as the entry point for the command line application.
+ * Based on this entrypoint you can call subcommands depending on your use case.
+ * Called in {@link cloud.foundry.cli.Main#main(String[])}
+ */
+@CommandLine.Command(name = "cf-control", header = "%n@|green Welcome to cf-control|@",
```

Comment:
You can import the CommandLine and just use the 
@Command

Reply:
Not sure what you mean. Mind to post an example diff? GitHub allows for this, it's the left-most "+-" button in the comment editor, above the text field.

Reply:
Alex meant: import picocli.CommandLine.Command; 
So i can just write: @Command.
Have changed it according to alex's suggestion.